RefTutorial.md

```diff
@@ -0,0 +1,61 @@
+# Running the F-Prime to OpenMCT Pipeline on the Ref Deployment
+In this tutorial we explore how to generate real time telemetry in OpenMCT on the Ref Deployment, which is a basic deployment present in F-Prime that is used to test basic functionality. 
+
+This tutorial assumes the user has installed OpenMCT, the F-Prime OpenMCT pipeline, as well as `fprime_gds` and `fprime_tools`, as detailed in `README.md`. 
+
+## Basic Setup
+First, we clone the devel branch of F-Prime.
+```
+git clone https://github.com/nasa/fprime.git
+```
+Navigate to the Ref Directory, and generate a build directory using `fprime-util`.
+```
+cd Ref
```

Comment:
Could we point users to SystemReference/HelloWorld tutorial or something instead? We don't want people to refer to this older reference anymore.

Reply:
I agree. I'm thinking I can add this in as part of https://github.com/nasa/fprime/issues/2162?